exercise1.py

```diff
@@ -0,0 +1,18 @@
+# File     : program1.py
+# Brief    :A program for simple arithmetic operations like add, subtract, multiplication, division and etc
+# Authore  : Deepti
+# Date     : 16-may-17
+a = input("enter a number for a \n")
+b = input("enter a number for b  \n")
+
+choice = raw_input( " enter the operation you want  \n")
+
+if choice == 'add':print a+b
+if choice == 'sub':print a-b
```

Comment:
Use Meaning full display at console using print, say dont print the output alone.
Print what is the output denotes + output